### PR TITLE
Add Loss Functions

### PR DESCRIPTION
**Pull request by soomiles**

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -36,9 +38,12 @@ def seed_everything(seed):
     num_epochs = 100
     lr = 1e-4
     lr_decay_step = 10
+    criterion_name = 'label_smoothing'
 
     train_log_interval = 20
-    name = "02_vgg"
+    # name = "02_vgg"
+    # set a name as time for debugging convenience
+    name = datetime.now(timezone(timedelta(hours=9))).strftime("%y%m%d_%H%M%S")
```
Comment (MoMentum99) [inline train.py:46]: 음 요거는 mode 를 따로두고 debug 모드이면 datetime 으로 가져오고 train 모드이면 원하는 name으로 하는 식으로 할까요?
argparse 이면 default 를 저걸로 주면 깔끔한데 지금은 하드코딩이라서 애매하긴 하네요

Comment (MoMentum99) [reply train.py:46]: 나머지는 다 좋은거 같습니당!

Comment (soomiles) [reply train.py:46]: 넵 혼동이 있을 수 있어서 지우는것도 좋아보이네요!
